examples/tracing-http-propagator/src/client.rs

```diff
@@ -66,5 +88,11 @@ async fn main() -> std::result::Result<(), Box<dyn std::error::Error + Send + Sy
     )
     .await?;
 
+    tracer_provider
```

Comment:
How do we feel about using anyhow in examples? On one hand it would be the typical way of doing this in the community, on the other it's an extra dep.

Reply:
neutral on that. I want to ensure we are consistent throughout the examples. (which we are not today, but I expect to get to that stage soon)